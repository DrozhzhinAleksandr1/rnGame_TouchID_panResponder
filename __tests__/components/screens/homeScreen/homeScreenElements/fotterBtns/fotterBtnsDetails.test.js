import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { FooterBtnDetails } from '../../../../../../app/components/screens/homeScreen/homeScreenElement/fotterbtns/footerBtnDetails';

it('create snapshot FooterBtnDetails', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<FooterBtnDetails btnDetailsText={'Details'}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});
