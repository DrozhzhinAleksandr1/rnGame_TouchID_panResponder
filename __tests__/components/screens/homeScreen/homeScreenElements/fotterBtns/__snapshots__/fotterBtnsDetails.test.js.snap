// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ErrorValidation 1`] = `
<TouchableWithoutFeedback>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#1D2E3E",
          "borderWidth": 0.5,
          "height": 40,
          "justifyContent": "center",
          "marginHorizontal": 5,
          "width": 120,
        },
        Object {
          "borderColor": "#5A6772",
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#5A6772",
          "fontFamily": "Roboto-Light",
          "fontSize": 16,
          "lineHeight": 19,
        }
      }
    >
      Details
    </Text>
  </Component>
</TouchableWithoutFeedback>
`;

exports[`create snapshot FooterBtnDetails 1`] = `
<TouchableWithoutFeedback>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#1D2E3E",
          "borderWidth": 0.5,
          "height": 40,
          "justifyContent": "center",
          "marginHorizontal": 5,
          "width": 120,
        },
        Object {
          "borderColor": "#5A6772",
        },
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#5A6772",
          "fontFamily": "Roboto-Light",
          "fontSize": 16,
          "lineHeight": 19,
        }
      }
    >
      Details
    </Text>
  </Component>
</TouchableWithoutFeedback>
`;
